modules/administration-guide/partials/proc_enabling-readiness-init-containers-for-olm-installer.adoc

```diff
@@ -0,0 +1,44 @@
+:_module-type: PROCEDURE
+
+[id="enabling-readiness-init-containers-for-olm-installer_{context}"]
+= Enabling readiness init containers for OLM installer
+
+To enable or disable readiness init containers for {prod-short} installed using the Operator Lifecycle Manager (OLM) installer:
+
+.Prerequisites
+
+* {prod-short} is installed with OLM installer.
+
+.Procedure
+
+. Find the {prod-short} Operator subscription name:
++
+[subs="+attributes,+quotes"]
+----
+$ {orch-cli} get subscriptions -n {prod-namespace}
+----
+
+. Find the CSV name in the {prod-short} Operator subscription:
++
+[source,shell,subs="+quotes,+attributes"]
+----
+$ {orch-cli} get subscription <subscription-name> -n {prod-namespace} -o yaml | grep installedCSV
+----
+
+. Edit the CSV:
++
+[source,shell,subs="+quotes,+attributes"]
+----
+$ {orch-cli} edit csv <csv-name> -n {prod-namespace}
```

Comment:
```suggestion
$ {orch-cli} edit csv _<csv-name>_ -n {prod-namespace}
```
If the admin/reader has to provide <csv-name>, then we need to italicize it according to:
https://redhat-documentation.github.io/supplementary-style-guide/#user-replaced-values
(And in that case, ideally we can tell the admin/reader where to find it.